packages/jest-emotion/src/index.js

```diff
@@ -69,14 +70,47 @@ function filterEmotionProps(props = {}) {
   return rest
 }
 
+function isShallowEnzymeElement(element, classNames) {
+  let renderedClassNames = (element.children || [])
+    .map(({ props = {} }) => props.className)
+    .filter(Boolean)
+    .join(' ')
+  return !classNames.some(className =>
+    new RegExp(className).test(renderedClassNames)
```

Comment:
`classNames` are just strings, so I think it would be better to just use a simple:
```js
renderedClassNames.includes(className)
```

Or create a regexp out of all class names with smth like `new RegExp(`(${classNames.join('|')})`)` and run this on `renderedClassNames`

Or even create an `areIntersecting` helper that could be used as follows:
```js
return !areIntersecting(classNames, renderedClassNames)
```